.changeset/sixty-melons-applaud.md

```diff
@@ -0,0 +1,8 @@
+---
+'@backstage/plugin-azure-sites-backend': minor
+'@backstage/plugin-azure-sites-common': minor
+'@backstage/plugin-azure-sites': minor
+---
+
+Implemented Permission framework to protect start and stop `Azure sites`
```

Comment:
You should probably call out in here in _what way_ that there are breakages, so people know what to do. As an adopter, I would like to be informed that I need to add `permissions: env.permissions` to the parameters of my router so I can compile.